packages/aila/src/protocol/schema.ts

```diff
@@ -38,6 +38,19 @@ export const MisconceptionSchema = z.object({
     .describe("Not to be used. Only included here for legacy support."), // There is some confusion here about what the LLM generates
 });
 
+export const MisconceptionSchemaWithoutLength = z.object({
```

Comment:
It would be useful to have a comment for each of these reminding us why they exist (we can't have min/max). In other places we've suffixed with `ForLLM` which does a similar thing

I wonder if it's worth writing a function to take a zod schema and replace the parts which aren't supported by the LLM automatically. That would need to be a follow up regardless